src/utils/index.js

```diff
@@ -187,11 +187,6 @@ export function getSegmentNumber(route, offset) {
   if (offset === undefined) {
     offset = currentOffset();
   }
-  for (let i = 0; i < route.segment_offsets.length; i++) {
-    if (offset >= route.segment_offsets[i]
-      && (i === route.segment_offsets.length - 1 || offset < route.segment_offsets[i + 1])) {
-      return route.segment_numbers[i];
-    }
-  }
-  return null;
-}
+
+  return Math.floor(offset / (60*1000));
```

Comment:
this isn't quite correct: log rotation bugs, missing segments, etc. however i think it's good enough for now

Reply:
It isn't currently listed in the [docs for derived data](https://api.comma.ai/#derived-data), but we recently started generating a file that gives you the first and last frame time of each segment relative to the start of the route named `frame_times.json`.

The file is similar to the GPS data points, but at the route level, for example:
```
https://chffrprivate.azureedge.net/chffrprivate3/v2/cb38263377b873ee/78392b99580c5920227cc5b43dff8a70_2017-06-12--18-51-47/frame_times.json
```